From stable-bounces@linux.kernel.org Thu Dec  8 15:26:50 2005
From: mchehab@brturbo.com.br
To: akpm@osdl.org
Date: Thu, 08 Dec 2005 21:18:47 -0200
Message-Id: <1134083966.7047.160.camel@localhost>
Cc: js@linuxtv.org, Mauro Carvalho Chehab <mchehab@brturbo.com.br>,
        linux-kernel@vger.kernel.org, mchehab@infradead.org,
        linux-dvb-maintainer@linuxtv.org, stable@kernel.org,
        Ricardo Cerqueira <v4l@cerqueira.org>
Subject: V4L/DVB (3135) Fix tuner init for Pinnacle PCTV Stereo

From: Ricardo Cerqueira <v4l@cerqueira.org>

- The Pinnacle PCTV Stereo needs tda9887 port2 set to 1
- Without this patch, mt20xx tuner is not detected and the board
  doesn't tune.

Signed-off-by: Ricardo Cerqueira <v4l@cerqueira.org>
Signed-off-by: Mauro Carvalho Chehab <mchehab@brturbo.com.br>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>


---
 drivers/media/video/saa7134/saa7134-cards.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- linux-2.6.14.3.orig/drivers/media/video/saa7134/saa7134-cards.c
+++ linux-2.6.14.3/drivers/media/video/saa7134/saa7134-cards.c
@@ -972,7 +972,7 @@ struct saa7134_board saa7134_boards[] = 
 		.radio_type     = UNSET,
 		.tuner_addr	= ADDR_UNSET,
 		.radio_addr	= ADDR_UNSET,
-		.tda9887_conf   = TDA9887_PRESENT | TDA9887_INTERCARRIER,
+		.tda9887_conf   = TDA9887_PRESENT | TDA9887_INTERCARRIER | TDA9887_PORT2_ACTIVE,
 		.inputs         = {{
 			.name = name_tv,
 			.vmux = 3,
